<launch>
<!-- include file="$(find robot_description)/launch/gazebo_sim.launch"/ -->
<node name="online_lqr" pkg="robot_control" type="online_lqr.py" output="screen"/>
<node name="imu_do" pkg="robot_control" type="imu_do_node.py" output="screen"/>
<node name="cmd_node" pkg="robot_control" type="cmd_node.py" output="screen"/>
<node name="key_cmd" pkg="robot_control" type="key_cmd.py"/>
<node name="multi_receive" pkg="robot_control" type="multi_receive.py"/>
<node name="control" pkg="robot_control" type="lqr_control1.py" output="screen"/>
<node name="odom" pkg="robot_control" type="odom_node"/>
<!--rviz-->
<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
<!--node pkg="rviz" type="rviz" name="rviz"/-->

<node pkg="rviz" type="rviz" name="rviz" args="-d $(find robot_control)/config/odom_test.rviz"/>
</launch>